tests/test_integration/test_dace_parsing.py

```diff
@@ -358,13 +365,47 @@ def numpy_stencil(inp: gtscript.Field[np.float64], outp: gtscript.Field[np.float
     )
 
     @dace.program
-    def call_frozen_stencil():
-        frozen_stencil(inp=inp, outp=outp)
+    def call_stencil():
+        numpy_stencil(
+            inp=inp, outp=outp, domain=(3, 3, 3), origin={"inp": (0, 0, 0), "outp": (0, 0, 0)}
+        )
 
     with pytest.raises(
         TypeError,
         match=re.escape(
             "Only dace backends are supported in DaCe-orchestrated programs." ' (found "numpy")'
         ),
     ):
-        call_frozen_stencil()
+        call_stencil()
+
+
+from gt4py.backend.dace_lazy_stencil import (  # noqa: E402 (needs to be guarded by above importorskip)
+    DaCeLazyStencil,
+)
```

Comment:
Is this here instead of at the top of the file for clarity, or is there another reason?

Reply:
missed that, I think.